src/main/java/com/spotifyapi/constant/ConstantTimeForCookie.java

```diff
@@ -0,0 +1,8 @@
+package com.spotifyapi.constant;
+
+public class ConstantTimeForCookie {
+
+    public static final int TIME_FOR_ACCESS_TOKEN_COOKIE = 60 * 60;
```

Comment:
Specify time in name (ONE_HOUR, ONE_WEEK, etc)

Reply:
Done!